doc/website/advanced/available-bindings-extensions.md

```diff
@@ -0,0 +1,9 @@
+# Available bindings and extensions
+
+The following bindings and extensions are available for iceoryx:
+
+| Name                | Description                                                                                                                 | Link                                                                                                                 |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| DDS gateway         | Gateway to the DDS protocol using [Cyclone DDS](https://cyclonedds.io/) or [FastDDS](https://github.com/eProsima/Fast-DDS)  | [https://github.com/eclipse-iceoryx/iceoryx-gateway-dds](https://github.com/eclipse-iceoryx/iceoryx-gateway-dds)        |
```

Comment:
@mossmaurice The gateway is only compatible with Cyclone DDS, or?

Reply:
No, there was a new contributor who added FastDDS support